src/test/java/command/TodoCommandTest.java

```diff
@@ -0,0 +1,25 @@
+package command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import henry.Task;
+import henry.TaskList;
+
+public class TodoCommandTest {
+
+    private static final String MESSAGE_SUCCESS = "OK. I ADDED THIS TASK TO MY LIST:\n\t\t\t %1$s.";
+
+    @Test
+    public void executeTest() {
```

Comment:
Could have a clearer test name? Nice comprehensive testing though.

Reply:
Modified in latest.